src/main/webapp/app/components/tabs/history-tab.store.tsx

```diff
@@ -26,7 +26,7 @@ export class HistoryTabStore {
     });
   }
 
-  setSelectedAuthor(author: { label: string; value: string }) {
+  setSelectedAuthor(author: { label: string; value: string } | null) {
```

Comment:
While `selectedAuthor` can be null, I don't want this method to be able to set it to null

Reply:
I'll fix it